src/map_queries.cpp

```diff
@@ -0,0 +1,43 @@
+#include <boost/algorithm/string/join.hpp>
+#include <boost/range/adaptor/transformed.hpp>
+
+#include "CLI/CLI.hpp"
+#include "query/queries.hpp"
+#include "spdlog/sinks/stdout_color_sinks.h"
+#include "spdlog/spdlog.h"
+
+using namespace pisa;
+
+int main(int argc, const char **argv)
+{
+    spdlog::drop("");
+    spdlog::set_default_logger(spdlog::stderr_color_mt(""));
+
+    std::string query_filename;
+    std::string terms_filename;
+    std::optional<std::string> stopwords_filename;
+    std::optional<std::string> stemmer = std::nullopt;
+    std::string separator = "\t";
```

Comment:
What sort of input are we expecting here? I thought our queries were formatted like:
```
123:hello abc test
```

Reply:
Wait, I think I follow now.

This program reads queries of the form:
```
123:hello abc test
```

And outputs them like
```
1\t2\t3
```
where `1,2,3` are the identifiers for `hello abc test` - right?

Reply:
Yes you are right.

I can add the query identifier too